script.js

```diff
@@ -28,6 +41,36 @@ const cardImage2 = document.getElementById('modelImg2');
 const descModal = document.getElementById('modalDes');
 const demo = document.getElementById('demoLink');
 const githubSrc = document.getElementById('githubSrc');
+const userName = document.getElementById('userName');
+const msg = document.getElementById('textareaMessage');
+const email = document.getElementById('email');
+
+// Form Data Local Storage
+
+function getFormInfo() {
+  const formData = {
+    name: userName.value,
+    message: msg.value,
+    email: email.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
+}
+
+function storeData() {
+  if (!localStorage.getItem('formData')) {
+    getFormInfo();
+  } else {
+    const fetchData = JSON.parse(localStorage.getItem('formData'));
+    userName.setAttribute('value', fetchData.userName);
+    email.setAttribute('value', fetchData.email);
+    msg.textContent = fetchData.msg;
+  }
+}
+window.onload = () => {
+  storeData();
+};
```

Comment:
- [ ] The expected behaviours of your for are:
     - After filling the form, and not submitting it, the entered values should be saved in the browser. :x:  After entering the values without submitting, I refreshed the browser, I expected the values to remain on the page, but disappeared. Kindly look into that.
     - When visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page. :x:  When I clicked the submit button after entering the values, I return to the page from formspree page and tried to reload, only the email that remain on the form the text in the textarea disappeared, the name input became undefined. Look at the below screen record to confirm what I am saying. Kindly look into it and fix the issue.
     - If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty. :heavy_check_mark:  

[your-page.webm](https://user-images.githubusercontent.com/26597432/204111052-7e536464-16a5-4c13-b776-fc3f18cfbe92.webm)